pomegranate/_utils.py

```diff
@@ -78,7 +78,7 @@ def _cast_as_parameter(value, dtype=None, requires_grad=False):
 
 
 def _update_parameter(value, new_value, inertia=0.0, frozen=None):
-	"""Update a parameters unles.
+	"""Update a parameter unles.
```

Comment:
I am not sure what the docstring should be here

Reply:
It seems perfectly clear. You update a parameter unles.